frontend/generateCard.js

```diff
@@ -1,65 +1,74 @@
 const cardHolder = document.getElementById('cardHolder');
 const cards = cardHolder.getElementsByClassName('card');
+let cardsData = {}
+let keywordData = [];
+let descriptionData = [];
 
-// Array of card titles
-const titles = [
-    'Deathtouch',
-    'Haste',
-    'Flying',
-    'Reach',
-    'Hexproof',
-    'Double Strike',
-    'First Strike',
-    'Lifelink'
-];
+// Fetch card data from API
+async function fetchCard() {
```

Comment:
Should this be called "fetchCardData" instead?

Reply:
Good idea